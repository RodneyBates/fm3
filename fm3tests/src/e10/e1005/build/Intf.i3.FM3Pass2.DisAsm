 ItkEOF()
 ItkLeftEnd()
 ItkInterfaceLt(L3(16_3),[1,0])
 ItkBlockDeclListLt(L1(16_1),[6,2])
 ItkTypeDeclLt([6,7])
 ItkDeclNo(D3,[6,7])
 ItkTypeDeclEquals([6,20])
 ItkReservedId(Rid"BOOLEAN",[6,22])
 ItkTypeDeclRt([6,7])
 ItkBlockDeclListRt(L1(16_1),[6,2])
 ItkInterfaceRt(L3(16_3),[1,0])
 ItkRightEnd()
 ItkBOF()
     0
