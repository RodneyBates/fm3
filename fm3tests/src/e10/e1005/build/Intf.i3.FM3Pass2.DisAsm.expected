 ItkEOF()
 ItkLeftEnd()
 ItkInterfaceLt(L4(16_4),[1,0])
 ItkDeclScopeLt(L1(16_1))
 ItkOpenScopeLt(L1(16_1))
 ItkOpenDeclListLt(L1(16_1),[6,2])
 ItkTypeDeclLt([6,7])
 ItkDeclNo(D4,[6,7])
 ItkTypeDeclType([6,20])
 ItkDefTopExprNo(L1(16_1),[6,22])
 ItkTypeDeclRt([6,7])
 ItkOpenDeclListRt(L1(16_1),[6,2])
 ItkOpenScopeRt(L1(16_1))
 ItkDeclScopeRt(L1(16_1))
 ItkInterfaceRt(L4(16_4),[1,0])
 ItkRightEnd()
 ItkBOF()
     0
